### PR TITLE
Add basic layout for new checkout page

### PR DESCRIPTION
**Pull request by lucasecdb**
#### What problem is this solving?
Adds the desktop layout for the new checkout page.

[Related clubhouse story](https://app.clubhouse.io/vtex/story/32812/layout-da-p%C3%A1gina-de-checkout-no-io)

#### How should this be manually tested?
[Workspace](https://chkio--checkoutio.myvtex.com/checkout)

#### Checklist/Reminders

- [x] Updated `README.md`.
- [x] Updated `CHANGELOG.md`.
- [x] Linked this PR to a Clubhouse story (if applicable).
- [ ] Updated/created tests (important for bug fixes).
- [ ] Deleted the workspace after merging this PR (if applicable).

#### Screenshots or example usage

image

#### Type of changes

✔️ | Type of Change
---|---
_ | Bug fix 
✔️ | New feature 
_ | Breaking change 
_ | Technical improvements

### CODE REVIEW

**Review by jeffersontuc — APPROVED** [review]

**Comment by jeffersontuc** [comment]
LGTM

File: react/PlaceOrder.css [hunk 1]
```diff
@@ -0,0 +1,3 @@
+.buttonWrapper :global(.vtex-button) {
+  font-size: 1rem !important;
```
Comment (darlenedms) [inline react/PlaceOrder.css:2]: ❗️ 👀

**Review by darlenedms — APPROVED** [review]

File: react/SummaryWrapper.tsx [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    },
+  } = useOrderForm()
+
+  const { insertCoupon } = useOrderCoupon()
+
+  return (
+    <div className="mh8-m mh0-l pt6 pt0-l bt b--muted-4 bn-l">
+      <ExtensionPoint
+        id="checkout-summary"
+        // @ts-ignore
```
Comment (marcoskwkm) [inline react/SummaryWrapper.tsx]: Could you add to the comment an explanation as to why this is necessary? Something like `// @ts-ignore: <name of the TS error>`

File: react/PlaceOrder.css (hunk 1 above)
Comment (lucasecdb) [reply react/PlaceOrder.css:2]: I talked with @davicosta99 and we will probably adjust this in styleguide :eyes:

**Review by marcoskwkm — APPROVED** [review]

File: react/SummaryWrapper.tsx (hunk 2 above)
Comment (lucasecdb) [reply react/SummaryWrapper.tsx]: sure! TIL

Comment (lucasecdb) [reply react/SummaryWrapper.tsx]: actually, using the prop `blockProps` enables me to remove this ts-ignore :-)

Comment (marcoskwkm) [reply react/SummaryWrapper.tsx]: Even better 👍

**Pull request merged**